### PR TITLE
OWC 2017 Finals + Conclusion newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
Bit of a different styling with this one, please give it a readover.

95% of the stuff is Evrien, 5% is me inserting/rewording things.

### CODE REVIEW

File: news/2017-12-20-owc-2017-concludes.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "osu! World Cup 2017 Concludes"
+date: 2017-12-20 10:00:00 +0000
+---
+
+Last weekend, the final circles were clicked, and the victors decided. 36 teams from across the globe whittled down to only 6, and from them, only 1 emerged victorious. After a hard-fought battle, **Poland** emerged victorious as the osu! Champions of 2017 - read on to find out more!
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: 32, not 36, unless I am mistaken in that it is 36 eligible teams.

File: news/2017-12-20-owc-2017-concludes.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+![](/wiki/shared/news/banners/owc_2017_banner.jpg)
+
+This year, 36 teams from across the globe entered and competed in the biggest circle-clicking spectacle that osu! has to offer. From new teams that just entered the fray like Romania to older returning ones like Japan, the tournament saw an overall increase in competitiveness from past years - joined with fiendishly difficult mappools to test skills of some of osu!'s best players.
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: 32, same as above

File: news/2017-12-20-owc-2017-concludes.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+The Finals Weekend saw 6 remaining teams – USA, Poland, UK, South Korea, Taiwan, and Germany – competing for the ultimate title of **osu! Champion**.
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: Not sure what determined the order of the countries here, but w/e

File: news/2017-12-20-owc-2017-concludes.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+In a decisive display of hard-won skill, **Poland** reigned supreme in the final match, taking home the osu! World Cup and the title until next year. If you supported them with a spiffy profile banner, you get to keep yours and broadcast your choice proudly till the next tournament starts!
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: Would it be better to say the next "official" tournament. Well... assuming MWC7K even happens lol

File: news/2017-12-20-owc-2017-concludes.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+This year, 36 teams from across the globe entered and competed in the biggest circle-clicking spectacle that osu! has to offer. From new teams that just entered the fray like Romania to older returning ones like Japan, the tournament saw an overall increase in competitiveness from past years - joined with fiendishly difficult mappools to test skills of some of osu!'s best players.
+
+The Finals Weekend saw 6 remaining teams – USA, Poland, UK, South Korea, Taiwan, and Germany – competing for the ultimate title of **osu! Champion**.
+
+In a decisive display of hard-won skill, **Poland** reigned supreme in the final match, taking home the osu! World Cup and the title until next year. If you supported them with a spiffy profile banner, you get to keep yours and broadcast your choice proudly till the next tournament starts! 
+
+Here's what went down throughout the final battles:
+
+# Finals Weekend
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: Could do with more images tbh.

File: news/2017-12-20-owc-2017-concludes.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+
+## South Korea vs Taiwan
+
+**South Korea: 7 - Taiwan: 6**
+
+Beginning with South Korea vs Taiwan, the endless rivalry between the two teams grew to a fever pitch, culminating in a colossal tiebreaker after some herculean effort from players on all sides. Pring and Pray from South Korea took turns carrying the team on their shoulders, while Flask and Rucker responded from Taiwan's side in kind.
+
+Poised to lose the tiebreaker-Tengaku in the wake of South Korea's earlier win against Brazil, Taiwan pulled out all the stops, sending Rucker in with a Hidden mod pick on the ultimate match of the set - scraping a hugely respectable score 300k, but still sadly losing the match overall to South Korea 7-6.
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: swap "score" and "300k", so "scraping a hugely respectalbe 300k score"

File: news/2017-12-20-owc-2017-concludes.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+
+Perhaps one of the most notable moments of the finals weekend, South Korea almost managed to scrape a win on AMAZING BREAK! with only 28,000~ score difference after a member of their team disconnected mid match. Outstanding effort.
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: tilda before 28k?

File: news/2017-12-20-owc-2017-concludes.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+
+Despite having been sent down to the Loser's Bracket by the Polish warriors in the weekend prior, the English roster recovered - knocking one decisive blow after another into the German defense line. Despite holding on to their best attempt, Dustice, imagaK, Neliel, and the rest could not muster up enough against the combined fury of Spare and Bubbleman – two of UK’s most prized circleclickers, ending the battle dominantly in favour of the United Kingdom 7-1.
+
+## United States vs Poland
+
+**Poland: 7 - United States: 6**
+
+Both strong contenders for the Champion title, the two teams felt tremendous pressure from each other. Team USA started off looking incredibly strong, taking 5 points in a row and producing FCs as if they were a cakewalk. 
+
+Just as everyone thought Team USA would finish the game in a flash and then walk on to claim their title 2 days later, idke’s tablet pen inexplicably broke, sending team United States' plan into disarray. Coupled with the wounded hand of their HardRock support player HappyStick, the US was suddenly painfully exposed to an explosive comeback from team Poland.
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: captial T for Team Poland

File: news/2017-12-20-owc-2017-concludes.md [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+
+And come back they did. Over the course of Poland’s subsequent performance, there were at least 2 instances where Team USA could have packed the match in but failed to due to either players failing or just being slightly under in score. Everyone held their breaths as Team Poland, upon the 12th map of the match – Kokou no Sousei, clenched the absolute tiniest margin of victory by a mere 182 points.
+
+At this moment, the American defense line crumbled, and then soon later the tiebreaker map went over to Poland. The Polish players celebrated as if they had just won the entire tournament, while the United States began the gruelling climb out of the Loser's Bracket.
+
+## United Kingdom vs South Korea 
+
+**United Kingdom: 7 - South Korea: 2**
+
+Returning to the Loser’s Bracket, team South Korea – the last hope for the Asian teams – was subjected to the wrath of team United Kingdom. Despite their amazing run earlier, the English gentlemen seemed to have abandoned the code for etiquette and simply decided upon a path of sheer, unadulterated dominance instead.
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: Captial T for Team UK

Comment (mangomizer) [reply news/2017-12-20-owc-2017-concludes.md]: Also the 2nd sentence sounds like it's coming out of a fanfic lmao

File: news/2017-12-20-owc-2017-concludes.md [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+
+Now, the stage was set.
+
+# Grand Finals
+
+## Poland vs United States (Match 1)
+
+**United States: 7 - Poland: 3**
+
+And finally, the stage was set for the most hyped match of the year – Grand Finals, Poland vs USA, and everything leading up to this point was just practice. On one hand, Team USA – the defending champion, reigning 2 years strong, with veterans who are well-rounded and new blood who are rising to step into the spotlight.
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: Team US, not USA?

File: news/2017-12-20-owc-2017-concludes.md [hunk 11]
```diff
@@ -0,0 +1,126 @@
...
+
+## Poland vs United States (Match 1)
+
+**United States: 7 - Poland: 3**
+
+And finally, the stage was set for the most hyped match of the year – Grand Finals, Poland vs USA, and everything leading up to this point was just practice. On one hand, Team USA – the defending champion, reigning 2 years strong, with veterans who are well-rounded and new blood who are rising to step into the spotlight. 
+
+Team Poland stood as the underdog despite being in the winner's bracket – having had so many close calls to winning an OWC title in the past, denied again and again by the victory of others.
+
+Team USA returned to the match-up with incredible momentum, having just locked down their match against Team UK the day prior. It was apparent from Game 1 that Team USA adjusted strategy to counter Team Poland more this time – banning Packet Hero, a No Mod map that Poland’s edge was just too sharp.
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: US as above?

Comment (mangomizer) [reply news/2017-12-20-owc-2017-concludes.md]: (twice)

File: news/2017-12-20-owc-2017-concludes.md [hunk 12]
```diff
@@ -0,0 +1,126 @@
...
+
+**United States: 7 - Poland: 3**
+
+And finally, the stage was set for the most hyped match of the year – Grand Finals, Poland vs USA, and everything leading up to this point was just practice. On one hand, Team USA – the defending champion, reigning 2 years strong, with veterans who are well-rounded and new blood who are rising to step into the spotlight. 
+
+Team Poland stood as the underdog despite being in the winner's bracket – having had so many close calls to winning an OWC title in the past, denied again and again by the victory of others.
+
+Team USA returned to the match-up with incredible momentum, having just locked down their match against Team UK the day prior. It was apparent from Game 1 that Team USA adjusted strategy to counter Team Poland more this time – banning Packet Hero, a No Mod map that Poland’s edge was just too sharp. 
+
+The American team captain Apraxia also showed willingness to become more active – with HappyStick still out of the picture, he had to step up and help his team stay afloat on certain picks.
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: not that it's wrong or anything, I just find the use of "american" a bit weird.

File: news/2017-12-20-owc-2017-concludes.md [hunk 13]
```diff
@@ -0,0 +1,126 @@
...
+
+Team Poland stood as the underdog despite being in the winner's bracket – having had so many close calls to winning an OWC title in the past, denied again and again by the victory of others.
+
+Team USA returned to the match-up with incredible momentum, having just locked down their match against Team UK the day prior. It was apparent from Game 1 that Team USA adjusted strategy to counter Team Poland more this time – banning Packet Hero, a No Mod map that Poland’s edge was just too sharp. 
+
+The American team captain Apraxia also showed willingness to become more active – with HappyStick still out of the picture, he had to step up and help his team stay afloat on certain picks. 
+
+Veteran players Wilchq, Rafis, and WubWoofWolf from team Poland  paid close attention to their opponents as they played, avoiding having 5 points taken in a row like they did in the beginning of the match 2 days ago.
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: double space between Poland and paid. Also captial T for Team Poland

File: news/2017-12-20-owc-2017-concludes.md [hunk 14]
```diff
@@ -0,0 +1,126 @@
...
+
+Toy from the United States stepped up and delivered a much-needed 3x100 FC on Made of Fire, eliciting undying praise from the spectating crowds. Unfazed by his incredible performance under the joint leadership of the Polish veterans WubWoofWolf and Piggey, the Polish mustered their best and stepped up to thwart the United States' advance.
+
+Before anyone knew it, Team Poland scored win after win and went from being down 2-4 to leading 6-5 - taking the game to the match point in their favour!
+
+The pick was Team USA’s as they were looking at tying the score before a necessary tiebreaker. Choosing between Ai wo Chikaishi Hime Kazari FM and Ryoushi no Umi no Lindwurm NM, they picked the latter – a highly challenging and gimmicky reading map reserved only for the top players. 
+
+Coming into the potentially final matchup, both teams seemed to have equal footing with Team USA possibly having a slight edge over the Polish. However, as everyone was dropping combos, WubWoofWolf – the one Polish player who has been around for every single OWC so far – stepped up and maintained combo through all hard sections.
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: Team US, not USA

File: news/2017-12-20-owc-2017-concludes.md [hunk 15]
```diff
@@ -0,0 +1,126 @@
...
+
+8 years of furious determination and unrelenting practice manifested themselves in a glorious instant, and in what is possibly one of the best performances ever seen in the osu! World Cup, WubWoofWolf scored the definitive FC to end all FCs on Lindwurm, firmly securing the win for the Polish side.
+
+And with it, the title and prize.
+
+<iframe src="https://clips.twitch.tv/embed?clip=BrightPatientFriseeSmoocherZ&autoplay=false&tt_medium=clips_embed" width="100%" height="360" frameborder="0" scrolling="no" allowfullscreen="true"></iframe>
+
+# Conclusion
+
+And that would conclude the final chapter to this year’s OWC. 36 amazing teams, one epic journey, and a year to remember – 2017 is the 10th anniversary of osu!, and together as a community we have traveled so far.
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: 32, not 36

File: news/2017-12-20-owc-2017-concludes.md [hunk 16]
```diff
@@ -0,0 +1,126 @@
...
+
+And with it, the title and prize.
+
+<iframe src="https://clips.twitch.tv/embed?clip=BrightPatientFriseeSmoocherZ&autoplay=false&tt_medium=clips_embed" width="100%" height="360" frameborder="0" scrolling="no" allowfullscreen="true"></iframe>
+
+# Conclusion
+
+And that would conclude the final chapter to this year’s OWC. 36 amazing teams, one epic journey, and a year to remember – 2017 is the 10th anniversary of osu!, and together as a community we have traveled so far. 
+
+**Congratulations to Team Poland, Team USA, and Team UK for their amazing run this year!** The talents they displayed, the prowess they demonstrated, from tears of regret to tears of joy – everything this year made for an amazing tournament.
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: USA or US I'm seriously doubting myself at this point

File: news/2017-12-20-owc-2017-concludes.md [hunk 17]
```diff
@@ -0,0 +1,126 @@
...
+What’s in store in the future? Nobody knows, and that’s what makes it exciting. As I was commentating the matches, many times I’ve thought to myself – "hey, a future OWC champion may be watching with us right now, and this may be his initiating moment!"
+
+I most certainly hope that I was right, and that the osu! community would venture farther – to the sky and the stars above.
+
+Stay gold, circleclickers, and remember – rhythm is just a click away!
+
+See you next time!
+
+—Evrien
+
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: Nothing about Loctav? rip.

File: news/2017-12-20-owc-2017-concludes.md (hunk 3 above)
Comment (mangomizer) [reply news/2017-12-20-owc-2017-concludes.md]: also USA?

File: news/2017-12-20-owc-2017-concludes.md [hunk 18]
```diff
@@ -0,0 +1,126 @@
...
+## United States vs Poland
+
+**Poland: 7 - United States: 6**
+
+Both strong contenders for the Champion title, the two teams felt tremendous pressure from each other. Team USA started off looking incredibly strong, taking 5 points in a row and producing FCs as if they were a cakewalk. 
+
+Just as everyone thought Team USA would finish the game in a flash and then walk on to claim their title 2 days later, idke’s tablet pen inexplicably broke, sending team United States' plan into disarray. Coupled with the wounded hand of their HardRock support player HappyStick, the US was suddenly painfully exposed to an explosive comeback from team Poland. 
+
+And come back they did. Over the course of Poland’s subsequent performance, there were at least 2 instances where Team USA could have packed the match in but failed to due to either players failing or just being slightly under in score. Everyone held their breaths as Team Poland, upon the 12th map of the match – Kokou no Sousei, clenched the absolute tiniest margin of victory by a mere 182 points.
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: USA?

File: news/2017-12-20-owc-2017-concludes.md [hunk 19]
```diff
@@ -0,0 +1,126 @@
...
+
+However, the United States' implacable momentum all but thrust them to a decisive early victory, placing Poland's hopes of a champion title once more into grave jeopardy with a 7-3 win in the United States' favour.
+
+## United States vs Poland (FINAL MATCH)
+
+**Poland: 7 - United States: 5**
+
+With only ten minutes of break time for the two teams to prepare, the agonizing wait was used by both sides to consider strategies and steel themselves for the fight ahead. Both teams were playing at their absolute strongest - the stakes had never been higher.
+
+Team Poland started off the second match with strategy adjustments this time – instead of banning Made of Fire HD, they banned Kakuzetsu Thanatos HR, a pick where Team USA just simply dominated. Meanwhile, Made of Fire, an "old but gold" AR8 map, really put a lot of good reading players to the test this time around, with players like Doomsday and WubWoofWolf showing signs of struggle, too.
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: USA

File: news/2017-12-20-owc-2017-concludes.md [hunk 20]
```diff
@@ -0,0 +1,126 @@
...
+
+Toy from the United States stepped up and delivered a much-needed 3x100 FC on Made of Fire, eliciting undying praise from the spectating crowds. Unfazed by his incredible performance under the joint leadership of the Polish veterans WubWoofWolf and Piggey, the Polish mustered their best and stepped up to thwart the United States' advance.
+
+Before anyone knew it, Team Poland scored win after win and went from being down 2-4 to leading 6-5 - taking the game to the match point in their favour!
+
+The pick was Team USA’s as they were looking at tying the score before a necessary tiebreaker. Choosing between Ai wo Chikaishi Hime Kazari FM and Ryoushi no Umi no Lindwurm NM, they picked the latter – a highly challenging and gimmicky reading map reserved only for the top players.
```
Comment (mangomizer) [inline news/2017-12-20-owc-2017-concludes.md]: USA?

File: news/2017-12-20-owc-2017-concludes.md [hunk 21]
```diff
@@ -0,0 +1,126 @@
...
+
+Toy from the United States stepped up and delivered a much-needed 3x100 FC on Made of Fire, eliciting undying praise from the spectating crowds. Unfazed by his incredible performance under the joint leadership of the Polish veterans WubWoofWolf and Piggey, the Polish mustered their best and stepped up to thwart the United States' advance.
+
+Before anyone knew it, Team Poland scored win after win and went from being down 2-4 to leading 6-5 - taking the game to the match point in their favour!
+
+The pick was Team United States’s as they were looking at tying the score before a necessary tiebreaker. Choosing between Ai wo Chikaishi Hime Kazari FM and Ryoushi no Umi no Lindwurm NM, they picked the latter – a highly challenging and gimmicky reading map reserved only for the top players.
```
Comment (MegaApplePi) [inline news/2017-12-20-owc-2017-concludes.md]: `’` -> `'` (there are 7 occurrences of this)

File: news/2017-12-20-owc-2017-concludes.md [hunk 22]
```diff
@@ -0,0 +1,126 @@
...
+
+And with it, the title and prize.
+
+<iframe src="https://clips.twitch.tv/embed?clip=BrightPatientFriseeSmoocherZ&autoplay=false&tt_medium=clips_embed" width="100%" height="360" frameborder="0" scrolling="no" allowfullscreen="true"></iframe>
+
+## Conclusion
+
+And that would conclude the final chapter to this year’s OWC. 32 amazing teams, one epic journey, and a year to remember – 2017 is the 10th anniversary of osu!, and together as a community we have travelled so far. 
+
+**Congratulations to Team Poland, Team United States, and Team Hnited Kingdom for their amazing run this year!** The talents they displayed, the prowess they demonstrated, from tears of regret to tears of joy – everything this year made for an amazing tournament.
```
Comment (MegaApplePi) [inline news/2017-12-20-owc-2017-concludes.md]: typo `Hnited Kingdom` -> `United Kingdom`

File: news/2017-12-21-owc-2017-concludes.md [hunk 23]
```diff
@@ -0,0 +1,126 @@
...
+
+## Finals Weekend
+
+### South Korea vs Taiwan
+
+**South Korea: 7 - Taiwan: 6**
+
+Beginning with South Korea vs Taiwan, the endless rivalry between the two teams grew to a fever pitch, culminating in a colossal tiebreaker after some herculean effort from players on all sides. Pring and Pray from South Korea took turns carrying the team on their shoulders, while Flask and Rucker responded from Taiwan's side in kind.
+
+Poised to lose the tiebreaker-Tengaku in the wake of South Korea's earlier win against Brazil, Taiwan pulled out all the stops, sending Rucker in with a Hidden mod pick on the ultimate match of the set - scraping a hugely respectable 300k score, but still sadly losing the match overall to South Korea 7-6.
```
Comment (osuShiro) [inline news/2017-12-21-owc-2017-concludes.md]: Repeating the final score is redundant and doesn't quite fit here
might wanna mention Yaong's ridiculous performance on Tengaku too

File: news/2017-12-21-owc-2017-concludes.md [hunk 24]
```diff
@@ -0,0 +1,126 @@
...
+
+Perhaps one of the most notable moments of the finals weekend, South Korea almost managed to scrape a win on AMAZING BREAK! with only ~28,000 score difference after a member of their team disconnected mid match. Outstanding effort.
+
+### United Kingdom vs Germany
+
+**United Kingdom: 7 - Germany: 1**
+
+Over the years the German roster has seen some monumental improvements and the birth of backbone carries like Dustice. The United Kingdom, fully determined in going for the very top, was not in the slightest interest of being stopped here. 
+
+Despite having been sent down to the Loser's Bracket by the Polish warriors in the weekend prior, the English roster recovered - knocking one decisive blow after another into the German defense line. Despite holding on to their best attempt, Dustice, imagaK, Neliel, and the rest could not muster up enough against the combined fury of Spare and Bubbleman – two of the United Kingdom's most prized circleclickers, ending the battle dominantly in favour of the United Kingdom 7-1.
```
Comment (osuShiro) [inline news/2017-12-21-owc-2017-concludes.md]: again, the final score is already given in the paragraph's title

File: news/2017-12-21-owc-2017-concludes.md [hunk 25]
```diff
@@ -0,0 +1,126 @@
...
+
+### United States vs Poland
+
+**Poland: 7 - United States: 6**
+
+Both strong contenders for the Champion title, the two teams felt tremendous pressure from each other. Team United States started off looking incredibly strong, taking 5 points in a row and producing FCs as if they were a cakewalk. 
+
+Just as everyone thought Team United States would finish the game in a flash and then walk on to claim their title 2 days later, idke's tablet pen inexplicably broke, sending Team United States' plan into disarray. Coupled with the wounded hand of their HardRock support player HappyStick, the US was suddenly painfully exposed to an explosive comeback from Team Poland. 
+
+And come back they did. Over the course of Poland's subsequent performance, there were at least 2 instances where Team United States could have packed the match in but failed to due to either players failing or just being slightly under in score. Everyone held their breaths as Team Poland, upon the 12th map of the match – Kokou no Sousei, clenched the absolute tiniest margin of victory by a mere 182 points.
```
Comment (osuShiro) [inline news/2017-12-21-owc-2017-concludes.md]: "upon the 12th map of the match – Kokou no Sousei,"
either use comma both ends, or dashes - you can't mix them in

File: news/2017-12-21-owc-2017-concludes.md [hunk 26]
```diff
@@ -0,0 +1,126 @@
...
+
+At this moment, the American defense line crumbled, and then soon later the tiebreaker map went over to Poland. The Polish players celebrated as if they had just won the entire tournament, while the United States began the gruelling climb out of the Loser's Bracket.
+
+### United Kingdom vs South Korea 
+
+**United Kingdom: 7 - South Korea: 2**
+
+Returning to the Loser's Bracket, Team South Korea – the last hope for the Asian teams – was subjected to the wrath of Team United Kingdom. Despite their amazing run earlier, the English gentlemen seemed to have abandoned the code for etiquette and simply decided upon a path of sheer, unadulterated dominance instead.
+
+Fighting bravely, Pring and Pray from the South Korean side stood up against the English menace as best they could with their signature backbone carry, but simply could not muster enough to claim the match, letting the United Kingdom run away with the match for a 7-2 finish in their favour, and securing themselves a place in the top 3 no matter what.
```
Comment (osuShiro) [inline news/2017-12-21-owc-2017-concludes.md]: stop repeating the final score lol

File: news/2017-12-21-owc-2017-concludes.md [hunk 27]
```diff
@@ -0,0 +1,126 @@
...
+
+### Poland vs United States (Match 1)
+
+**United States: 7 - Poland: 3**
+
+And finally, the stage was set for the most hyped match of the year – Grand Finals, Poland vs United States, and everything leading up to this point was just practice. On one hand, Team United States – the defending champion, reigning 2 years strong, with veterans who are well-rounded and new blood who are rising to step into the spotlight. 
+
+Team Poland stood as the underdog despite being in the winner's bracket – having had so many close calls to winning an OWC title in the past, denied again and again by the victory of others.
+
+Team United States returned to the match-up with incredible momentum, having just locked down their match against Team United Kingdom the day prior. It was apparent from Game 1 that Team United States adjusted strategy to counter Team Poland more this time – banning Packet Hero, a No Mod map that Poland's edge was just too sharp.
```
Comment (osuShiro) [inline news/2017-12-21-owc-2017-concludes.md]: too sharp on ? I feel like the sentence is missing a word

File: news/2017-12-21-owc-2017-concludes.md (hunk 23 above)
Comment (Ephemeralis) [reply news/2017-12-21-owc-2017-concludes.md]: redundancy acceptable here due to the fact that the entire paragraph is one encoded block of information, removing the scores makes it harder to ascertain lead without backtracking to beginning of paragraph (which most people wont)

**Pull request merged**